docs/shared/versioning.md

```diff
@@ -0,0 +1,25 @@
+# Android Versioning
+
+This repository uses the same `version.txt` to define the version for Android Components, Focus, and Fenix. The following formatting corresonds to release branches 
+on the release track.
+
+## Format
+
+XXX.0 (where XXX is defined as the current desktop release #)
+
+*Please note: the initial release is `major.minor`, but the following releases will be `major.minor.patch`*
+
+* Levaraging semantic versioning, the first and second digit will be directly tied to the associated desktop release 
```

Comment:
s/levaraging/leveraging

But in general, I'm a bit nervous about saying semantic versioning here since AFAIK it really isn't (and it carries a very-particular connotation with it)